README.md

```diff
@@ -138,10 +150,10 @@ Next, use `prepare-extracts.py` to generate the `osmium extract`
 configuration file for the 6 slices. For instance:
 
 ```
-python scripts/prepare-extracts.py europe-latest.json --width 3 --height 2
+python prepare-extracts.py tanzania-latest.json --width 3 --height 2
```

Comment:
This script currently takes a single argument of `ratio` and produces `ratio^2` smaller bounding boxes
```
.......      .......
|     |      |  |  | 
|     |  =>  .......
|     |      |  |  |
```````      ```````
```